helix-term/src/keymap.rs

```diff
@@ -35,10 +35,10 @@ use std::collections::HashMap;
 //          f = find_char()
 //          g = goto (gg, G, gc, gd, etc)
 //
-//          h = move_char_left(n)
-//          j = move_line_down(n)
-//          k = move_line_up(n)
-//          l = move_char_right(n)
+//          h = move_char_left(n)   || arrow-left  = move_char_left(n)
```

Comment:
You don't have to update this comment, but you can add these to https://github.com/helix-editor/helix/blob/master/book/src/keymap.md :)

Reply:
I imagine in the future view-mode would probably also have arrow keys with similar functionality to j and k included.